### DISCUSSION TITLE
Todos discussion

### DISCUSSION THREAD

**Discussion by danymat** [nvim-neorg/neorg#43] [Ideas]
Hello @vhyrro , thank you again for your huge contribution to the nvim community.

I'm in the process of creating a todo.txt Neovim plugin in order to manage todos, but I'm still wondering if it's not better to contribute to the GTD and Neovim task addicts in Neorg itself. 

I saw that you want to better manage todos, so I'm wondering what's your point of view regarding my idea: is it better to create a plugin for the todo.txt plugin for the specifications (https://github.com/todotxt/todo.txt) or contributing to the todos development of Neorg? I'm open to contribute to the Neorg core, I just need to know what are your plans regarding this part of the work (todos development).

**Comment by vhyrro** [comment]
Totally up to you man, it all depends on what you're more comfy doing. Even though in theory GTD is planned for 0.2 there's totally nothing stopping anyone who feels like it from starting work on it earlier and fleshing it out. It'll simply make its way into 0.2 faster :)

I can't make everything "perfect" and so I don't think I'll manage to make an entire GTD and zettelkasten system without a serious chunk of time lol

> I just need to know what are your plans regarding this part of the work

Are you talking workflow-related ideas? Syntax based ideas? Or do you just mean "do you have anything very specific that I should account for"?

**Reply by danymat** [reply]
> Totally up to you man, it all depends on what you're more comfy doing. Even though in theory GTD is planned for 0.2 there's totally nothing stopping anyone who feels like it from starting work on it earlier and fleshing it out. It'll simply make its way into 0.2 faster :)

I can contribute to these features, the first reason being that I'm desperate to use GTD in neorg and nvim ! The second one is to learn lua plugin development for nvim, and the fact that you have a good documentation and a well structured code base moves me to contribute to this project 

> I can't make everything "perfect" and so I don't think I'll manage to make an entire GTD and zettelkasten system without a serious chunk of time lol

These 2 systems are literally the systems I use in everyday life, so I'm very happy to help in the specs and development 

> > I just need to know what are your plans regarding this part of the work
> 
> 
> 
> Are you talking workflow-related ideas? Syntax based ideas? Or do you just mean "do you have anything very specific that I should account for"?

I was actually referring to any views you want to share about these 2 developments to someone who wants to contribute to yhe project.

Btw do the contributors have a group chat ?

**Comment by vhyrro** [comment]
For those interested the conversation continued on the discord. I don't wanna seem like I just ghosted dany lol
